charts/shoot-core/components/charts/coredns/templates/coredns-deployment.yaml

```diff
@@ -104,7 +105,11 @@ spec:
             path: /ready
             port: 8181
             scheme: HTTP
-          initialDelaySeconds: 5
+          initialDelaySeconds: 30
+          periodSeconds: 10
+          timeoutSeconds: 5
```

Comment:
Why did you increase the `timeoutSeconds` to `5`? If the dns reacts only after 5 seconds clients will usually already have entered failure state. My understanding is that usual dns timeout is rather in the range of 2 seconds. I do not understand why the readiness probe should take longer.
However, I see that it is specified like that for the liveness probe, though.

Reply:
For overloaded replicas 1 second might be too low and the replica will be very often added and removed from the backend serving pool. For other type of applications I would use something like 30 seconds, but I also have the gut feeling that this will be too high value for a DNS server. I haven't observed any issues because of the `1s` timeout, so I am fine to reduce it. Do you have any hard opinion what should be the timeout value?

Reply:
No, not really. I was just thinking about a dns issue @DockToFuture and myself saw where some of the dns instances were not answering properly (or in the desired timeframe). The instances were ready all the time, but only a restart resolved the issue.

@DockToFuture What do you think?

Reply:
I would increase it to not more than two seconds. With a larger timeout we will see dns resolution delays in case the request could not be resolved. And two seconds should be more than enough for a dns query.

Reply:
Thanks for the feedback, I have decreased it back to 1 second.

Reply:
What do you think to also decrease `failureThreshold` to `1`? Now it has to fail three times in row to let k8s exclude the pod from the service backend.

Reply:
Indeed reducing the ´failureThreshold` might be a good idea. I am a bit wary, though, that `1` might cause a bit too much issues. However, we can give it a try.

Reply:
I just checked probes metrics on one node of a relatively large shoot cluster where coredns pod is running since some time
```bash
$ curl -H "Authorization: Bearer $TOKEN" https://localhost:10250/metrics/probes -ksS | grep coredns
prober_probe_total{container="coredns",namespace="kube-system",pod="coredns-7767b5bf6-tgvzx",pod_uid="a61149c3-c3a1-48fd-94f8-034bdbb9cb19",probe_type="Liveness",result="successful"} 5850
prober_probe_total{container="coredns",namespace="kube-system",pod="coredns-7767b5bf6-tgvzx",pod_uid="a61149c3-c3a1-48fd-94f8-034bdbb9cb19",probe_type="Readiness",result="successful"} 5856
```
In the last ~16 hours there is no probe failure.

Reply:
Timeout is now set to 2 seconds and the failureThreshold to 1. Please, take a look again.